samza-core/src/main/java/org/apache/samza/container/host/PosixCommandBasedStatisticsGetter.java

```diff
@@ -86,28 +62,17 @@ private long getTotalPhysicalMemory() throws IOException {
     List<String> processIds = getAllCommandOutput(new String[]{"sh", "-c", "pgrep -P $PPID"});
```

Comment:
nit: rename method to getTotalPhysicalMemoryUsageBytes
so that the purpose and the unit of the response is clear to the caller.